triplea_maps.yaml

```diff
@@ -2273,3 +2273,12 @@
     Edition.</p>
     <br><p>Key features include combining UK Pacific and ANZAC into one nation, revised
     order of play for nations, victory objectives and 8 rounds of play.</p>
+- mapName: Zombies-World War 2
+  mapCategory: EXPERIMENTAL
+  url: https://github.com/triplea-maps/zombies-World_War_2/archive/master.zip
+  version: 1
+  img: https://raw.githubusercontent.com/triplea-maps/zombies-World_War_2/master/preview.png
```

Comment:
This URL leads to a 404. 
Navigating to the image in the repo, clicking download and copying that link, the URL is: https://github.com/triplea-maps/zombies-World_War_2/raw/master/Preview.png
The difference is the 'P' vs 'p'. A lower case 'p' is preferred, renaming the image would be a preferred fix.

Reply:
Fixed

Reply:
Re-ran checks, they passed now  :+1: